tpm2_pytss/constants.py

```diff
@@ -1267,6 +1310,16 @@ class TPMA_CC(TPM_FRIENDLY_INT):
     RES_MASK = lib.TPMA_CC_RES_MASK
     RES_SHIFT = lib.TPMA_CC_RES_SHIFT
 
+    @property
+    def commandindex(self) -> int:
+        """int: The command index"""
+        return self & self.COMMANDINDEX_MASK
+
+    @property
+    def chandles(self) -> int:
```

Comment:
do we want:

```python
def is_flushed()
def is_vendor()
def is_nv()
def is_extensive()
def has_rhandle()
... etc ...
```

Reply:
Possibly, could be useful with the different bits of TPMA_NV and TPMA_OBJECT for example